pkg/vpcmodel/routing_paths.go

```diff
@@ -44,6 +46,37 @@ func (p Path) Empty() bool {
 	return len(p) == 0
 }
 
+// ConcatPaths returns a new path concatenating the passed in paths
+func ConcatPaths(paths ...Path) Path {
+	return slices.Concat(paths...)
+}
+
+func ConcatWithResource(resource VPCResourceIntf, path Path) Path {
```

Comment:
That is more like PrependResource.
Also, why not making this function a method of `Path`?

Reply:
right, changed